code/controllers/subsystems/lighting.dm

```diff
@@ -50,6 +50,8 @@ SUBSYSTEM_DEF(lighting)
 	for (var/turf/T in (targets || world))
 		if (T.dynamic_lighting && T.loc:dynamic_lighting)
 			T.lighting_build_overlay()
+		// If this isn't here, BYOND will set-background us.
+		CHECK_TICK
```

Comment:
is this the cause of the startup delay?

Reply:
> is this the cause of the startup delay?

No. I ported this because this one of part fixes.